### ISSUE TITLE
Error on Ubuntu 18.4.2 , gnome 3.28.2

### ISSUE THREAD

**Issue by TobiHoKage** [BlinkBP/walkpaper#13]
After the recent update on extensions.gnome.org, walkpaper stopped working, and upon trying to reinstall it, it returns an error.

**Comment by Eren-is** [comment]
Thank you for bringing this up!
It was my mistake when publishing the newest version. It is not compatible with GNOME versions lower than 3.30. I disabled that update on extensions.gnome.org and sent a correct one for review.
I am not sure if you will be able to update (in fact downgrade) through extensions.gnome.org right now. If not you can download the pre3.30 release from here and install the zip using GNOME Tweaks.

**Comment by TobiHoKage** [comment]
I found a workaround couple of minutes after I posted here, I disabled "extensions" using Tweak tools, went to extentnsions.gnome.org, I reinstalled Walkpaper, and from setting there I set my backgrounds, opened tweak tools, enabled "extensions" , and it started working again. Why? I don't know

Regarding the downgrade, I have no clue. The issue got solved for me regardless, didn't see the notification till now!

Thanks anyway!
